### PR TITLE
Close #1481 verify if set onlye scale

### LINKED ISSUE #1481: Precision and Scale bug mysql
When I create table with column decimal, for example:

```php
->addColumn(
                'id_usuario_inclusao', 
                'decimal', [
                    'default' => null,
                    'null' => false,
                    'precision' => 30,
                    'scale' => 0
                ])
```

the method protected function getColumnSqlDefinition() in  Phinx\Db\Adapter\MysqlAdapter, no set precision and scale because the verification `if ($column->getPrecision() && $column->getScale()) {` not works.

If I set scale how true, 1 and more it works, for example:
```php
->addColumn(
                'id_usuario_inclusao', 
                'decimal', [
                    'default' => null,
                    'null' => false,
                    'precision' => 30,
                    'scale' => true
                ])
```
My suggest is change the verification `if` to:

```php
 if ($column->getPrecision() && $column->getScale()) {
            $def .= '(' . $column->getPrecision() . ',' . $column->getScale() . ')';
        } elseif ($column->getScale()) {
            $def .= '(' . 10 . ',' . $column->getScale() . ')';
        } elseif ($column->getPrecision()) {
            $def .= '(' . $column->getPrecision() . ',' . 0 . ')';
        }
```

### PR DESCRIPTION
**Pull request by HDias**
Verification if set only Scale to MysqlAdapter

### CODE REVIEW

File: src/Phinx/Db/Adapter/MysqlAdapter.php [hunk 1]
```diff
@@ -1117,8 +1117,15 @@ protected function getColumnSqlDefinition(Column $column)
             $sqlType = $this->getSqlType($column->getType(), $column->getLimit());
             $def = strtoupper($sqlType['name']);
         }
+        
         if ($column->getPrecision() && $column->getScale()) {
```
Comment (garas) [inline src/Phinx/Db/Adapter/MysqlAdapter.php]:
```suggestion
        if ($column->getPrecision()) {
```

It should be only precision check here, so it works with scale set to any number, 0, or not set.

I gave correct condition in first comment, but I made this mistake in second comment https://github.com/cakephp/phinx/issues/1481#issuecomment-454351168

**Comment by HDias** [comment]
I found de the problem, will send new pull request

**Comment by garas** [comment]
Add commits to the same branch `patch-2` of your fork and they will appear here. You don't need open new PR.

**Comment by dereuromark** [comment]
Any chance we can have a small test case?

**Comment by dereuromark** [comment]
ping @HDias

**Comment by AgelxNash** [comment]
```

**Comment by dereuromark** [comment]
ping

**Comment by othercorey** [comment]
Closing due to no activity.

**Pull request closed without merging**